py-scripts/lf_json_util.py

```diff
@@ -1,4 +1,4 @@
-
```

Comment:
No header at all in this file?

Reply:
Guess not. Can add to the list for follow-up canges (outside of this PR).